### DISCUSSION TITLE
Cleaning up Backup Repository

### DISCUSSION THREAD

**Discussion by hasantahir** [glushchenko/fsnotes#1393] [Q&A]
Having set the backup frequency to 1 hour without much thought, the Repo file size has now swollen to ~15 GB. It is now a few minutes for FS Notes to start up. Is there any in-built way to clean up the Git repo?

Thanks

**Comment by Evanavevan** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，你发的邮件我已经收到，我会尽快阅读你的邮件的
